.github/ISSUE_TEMPLATE/RELEASE.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Release
+about: Schedule a release
+---
+
+## Main Release
+
+Released quarterly on January 2nd, April 1st, July 1st and October 15th.
+
+* [ ] Open a release ticket e.g. https://github.com/python-pillow/Pillow/issues/3154
+* [ ] Develop and prepare release in `main` branch.
+  * [ ] Add release notes for 11.2.1 https://github.com/python-pillow/Pillow/pull/8885
+* [ ] Check [GitHub Actions](https://github.com/python-pillow/Pillow/actions) to confirm passing tests in `main` branch.
+* [ ] Check that all the wheel builds pass the tests in the [GitHub Actions "Wheels" workflow](https://github.com/python-pillow/Pillow/actions/workflows/wheels.yml) jobs by manually triggering them.
+* [ ] In compliance with [PEP 440](https://peps.python.org/pep-0440/), update version identifier in `src/PIL/_version.py`
+* [ ] Run pre-release check via `make release-test` in a freshly cloned repo.
+* [ ] Create branch and tag for release e.g.:
+  ```bash
+  git branch [[MAJOR.MINOR.PATCH]]
+  git tag [[MAJOR.MINOR.PATCH]]
```

Comment:
The version of this in RELEASING.md [is](https://github.com/python-pillow/Pillow/blob/main/RELEASING.md#main-release)
```
git branch 5.2.x
git tag 5.2.0
```
Rather than .x and .0, you've made both the same here.

We should keep .x, and since this is a 'Main release', the patch number should be zero.

```suggestion
  git branch [[MAJOR.MINOR]].0
  git tag [[MAJOR.MINOR]].0
```